app/controllers/main_routes/main_routes.py

```diff
@@ -112,7 +84,59 @@ def downloadSupervisorPortalResults():
     )
     return send_file(excel.relativePath, as_attachment=True, attachment_filename=excel.relativePath.split('/').pop())
 
-@main_bp.route('/lsf/<formHistoryId>/submitToBanner', methods=['GET'])
+@main_bp.route('/supervisorPortal/liveSearch', methods=['GET'])
+def SupervisorPortalSearch():
+    """
+    Returns a list of users that match a given string
+    """
+    def searchSupervisorPortal(searchType, userInput):
```

Comment:
I'm not a fan of the method being defined inside another method approach here. This should go in logic/search.py